libs/api/domains/syslumenn/src/lib/models/dataUpload.ts

```diff
@@ -0,0 +1,32 @@
+export type DataUpload = {
+  id: string
+  data: {
+    id: string
+    name: string
+    person: [Person]
+    attachments: [Attachment]
```

Comment:
Is this intentional? Array of one `Person` and one `Attachment`.

Reply:
🤦  nice catch 👏